### PR TITLE
Add Protobuff docs

### PR DESCRIPTION
**Pull request by owentmfoo**

### CODE REVIEW

File: docs/open_streaming_architecture/stream_api/stream_client.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ These libraries can be found at NuGet packages on the [McLaren Applied NuGet fee
 
 Given the API is created under the gRPC framework, it is possible to generate client and server classes in a language of
 your choice. 
-For details documentation of each of the RPC calls, refer to the [proto files](https://github.com/orgs/mat-docs/packages/nuget/package/MA.Streaming.API.Proto).
 
+For details documentation of each of the RPC calls, refer to the [protocol documentation](docs.md).
```
Comment (TimOliverMAT) [inline docs/open_streaming_architecture/stream_api/stream_client.md:16]: Should be either "details" or "documentation", not "details documentation"

File: mkdocs.yml [hunk 2]
```diff
@@ -49,11 +49,13 @@ nav:
       - Overview:                     open_streaming_architecture/index.md
       - Stream API:
         - Overview:                   open_streaming_architecture/stream_api/index.md
-        - Stream Client API:          open_streaming_architecture/stream_api/stream_client.md
-        - Stream Server API:          open_streaming_architecture/stream_api/stream_server.md
-        - Key Generation Service:       open_streaming_architecture/key_generation.md
+        - Stream API Client:          open_streaming_architecture/stream_api/stream_client.md
+        - Stream API Server:          open_streaming_architecture/stream_api/stream_server.md
+        - Key Generation Service:     open_streaming_architecture/key_generation.md
```
Comment (TimOliverMAT) [inline mkdocs.yml:54]: I think you changed this in the previous PR so does the link updating?

Comment (owentmfoo) [reply mkdocs.yml]: updated

File: docs/open_streaming_architecture/stream_api/stream_client.md (hunk 1 above)
Comment (owentmfoo) [reply docs/open_streaming_architecture/stream_api/stream_client.md]: updated

**Review by TimOliverMAT — APPROVED** [review]

**Pull request merged**